zproject/urls.py

```diff
@@ -243,6 +243,9 @@
         {'POST': 'regenerate_api_key'}),
     url(r'^users/me/enter-sends$', 'rest_dispatch',
         {'POST': 'change_enter_sends'}),
+    url(r'^users/me/star_user$', 'rest_dispatch',
+        {'GET': 'list_starred_user',
+         'POST': 'mark_user_star'}),
```

Comment:
the POST would be more appropriately a PUT.

@timabbott: Are we encouraging more RESTful endpoints, and if so, when are we using method names ending in `_backend`?
Do my comments on naming in actions/ and views/ make sense? I don't see anything in readthedocs, but I believe that do_foo in lib/actions.py actually modified the DB, whereas things in views/ don't modify the DB but instead call out to do_foo methods to make those changes.